src/include/gdb_packet.h

```diff
@@ -70,11 +70,15 @@ typedef struct gdb_packet {
 
 /* GDB packet transmission configuration */
 void gdb_set_noackmode(bool enable);
+bool gdb_noackmode(void);
 
 /* Raw GDB packet transmission */
 gdb_packet_s *gdb_packet_receive(void);
 void gdb_packet_send(const gdb_packet_s *packet);
 
+void gdb_packet_ack(bool ack);
+bool gdb_packet_get_ack(const uint32_t timeout);
```

Comment:
The `const` here isn't doing anything (it's fine and welcome in the function definition, but isn't useful in this declaration), please drop it.

Reply:
copy paste :)